### PR TITLE
verify trigger fire responds with any successful status code

### PR DESCRIPTION
**Pull request by abaruni**

### CODE REVIEW

**Comment by csantanapr** [comment]
@jasonpet could you review this PR

File: provider/consumer.py [hunk 1]
```diff
@@ -353,8 +353,12 @@ def __fireTrigger(self, messages):
 
                     # Manually commit offset if the trigger was fired successfully. Retry firing the trigger
                     # for a select set of status codes
-                    if status_code == 200:
-                        logging.info("[{}] Fired trigger with activation {}".format(self.trigger, response.json()['activationId']))
+                    if status_code in range(200, 300):
```
Comment (csantanapr) [inline provider/consumer.py]: that might be 299 not 300 if range is inclusive

Comment (abaruni) [reply provider/consumer.py]: the lower bound is inclusive while the upper bound is exclusive. i checked 😉

Comment (csantanapr) [reply provider/consumer.py]: haha !

**Pull request merged**